src/main/resources/logback-spring.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <configuration>
-  <springProperty name="OTEL_SERVICE_VERSION" source="build.version"/>
-  <appender name="ECS_JSON_CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
-    <encoder class="co.elastic.logging.logback.EcsEncoder">
-      <serviceName>${OTEL_SERVICE_NAME:-undefined}</serviceName>
-      <serviceVersion>${OTEL_SERVICE_VERSION}</serviceVersion>
-      <serviceEnvironment>${OTEL_SERVICE_ENVIRONMENT:-undefined}</serviceEnvironment>
-    </encoder>
-  </appender>
-  <root level="INFO">
-    <appender-ref ref="ECS_JSON_CONSOLE"/>
-  </root>
+    <springProperty name="OTEL_SERVICE_VERSION" source="build.version"/>
+    <appender name="ECS_JSON_CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder class="co.elastic.logging.logback.EcsEncoder">
+            <serviceName>${OTEL_SERVICE_NAME:-undefined}</serviceName>
+            <serviceVersion>${OTEL_SERVICE_VERSION}</serviceVersion>
+            <serviceEnvironment>${OTEL_SERVICE_ENVIRONMENT:-undefined}</serviceEnvironment>
+        </encoder>
+    </appender>
+    <root level="${DEFAULT_LOGGING_LEVEL}">
```

Comment:
what about refactoring this system property to root logging level? what do you think?

Reply:
Working on it

Reply:
Changed it 13e81b9c9fa90736e282d996278f51f9622591a1